po/LINGUAS

```diff
@@ -1,17 +1,18 @@
 bg_BG
+ca
 cs
 da
 de
 es
+fr
 hu
 id
 it
 nl
-pt_BR
-sv
+oc
```

Comment:
if it's not much asking, could you fix the sorting in one commit and add the oc entry together with the second commit? Once the file actually exists.

Reply:
I don't how to do this =/

Reply:
`git rebase -i [--autostash --autosquash] $(git merge-base HEAD master)` and the `edit` the commit. See *SPLITTING COMMITS* in `man 1 git-rebase`.

Reply:
I did everything via the user interface ^^
If I clone my fork and run this line, will it be enough? I'm really not into git :S

Reply:
:(